.github/workflows/test.yml

```diff
@@ -47,12 +44,21 @@ jobs:
       - name: Enable Rust Caching
         uses: Swatinem/rust-cache@v2
 
-      - name: Test
+      - name: Build
         # Build test binary with `testing` feature, which requires `hotshot_example` config
         run: |
           export RUSTFLAGS="$RUSTFLAGS --cfg hotshot_example"
           export PATH="$PWD/target/release:$PATH"
           cargo build --locked --bin diff-test --release
-          cargo test --locked --release --workspace --all-features --no-run
-          cargo test --locked --release --workspace --all-features --verbose -- --test-threads 1 --nocapture
+          cargo nextest run --locked --release --workspace --all-features --no-run
         timeout-minutes: 90
+
+      - name: Test
+        env:
+          CARGO_TERM_COLOR: always
+        # Build test binary with `testing` feature, which requires `hotshot_example` config
+        run: |
+          export RUSTFLAGS="$RUSTFLAGS --cfg hotshot_example"
+          export PATH="$PWD/target/release:$PATH"
+          cargo nextest run --locked --release --workspace --all-features --retries 2 --verbose -E '!test(slow_)'
```

Comment:
I think it would be nice to add just recipies to mirror the new test commands locally since it's also nice to run the tests quickly locally. And sometimes we want to re-run the same tests as in a CI job locally.

Reply:
a7d14fdaa48aecfe46083c956658bf89dbcb4a85